web/modules/custom/dpl_opening_hours/src/Plugin/rest/resource/v1/OpeningHoursResource.php

```diff
@@ -37,8 +37,8 @@ public function getPluginDefinition(): array {
           Request::METHOD_GET => [
             'branch_id' => [
               'name' => 'branch_id',
-              'type' => 'integer',
-              'description' => 'The id of the branch for which to retrieve opening hours.',
+              'type' => 'string',
+              'description' => 'The id(s) of the branch(es) for which to retrieve opening hours. Can be a single id or a comma-separated list of ids.',
```

Comment:
Consider if the specification can be updated to properly specify the format.

The specification actually supports `array` types with a separate entry `items` for the value and `collectionFormat` for the delimiter.